### ISSUE TITLE
dump annotation error - constructing empty shapely linestring 

### ISSUE THREAD

**Issue by nmaxwell** [opencv/cvat#1336]
I'm getting this error when I try to dump my annotations:

Can not dump annotations for the task. Code: 500. Message: "cvat.apps.engine.utils.InterpreterError: ValueError at line 90: LineStrings must have at least 2 coordinate tuples\n"

strange thing is my annotations only have points, no polylines, so not sure why linestrings are being made. I'm a bit worried that I won't be able to export my work, that it will be lost.

attaching the docker log 
[cvat-log.txt](https://github.com/opencv/cvat/files/4406400/cvat-log.txt)

thanks!

**Comment by zhiltsov-max** [comment]
Hello. Could you please describe what annotations does the task have and what is your version of CVAT? It looks like there are some interpolation mode shapes (tracks), which can only be bounding boxes. Is is possible that you've imported annotations?

**Comment by nmaxwell** [comment]
Hi,

Thanks for the response. I've been using 'Create Shape' to add 'Points' annotations, and then using 'Merge Shapes' to merge them into one track, across multiple frames, and setting a single attribute for the track.

I think there were instances where I implicitly interpolated, since I merged shapes across images that didn't have any shapes. Eg. add shape to image 50, add shape to image 55, merge them; this then auto-adds shapes to images 51, 52, 53, 54, which I then adjust. Maybe that counts as interpolation?

Now as for the version, I pulled develop originally on 3/19/2020, which is the version that gave the above error. 

Last night, I pulled develop again, and did a 'docker-compose build'. I now get a different error:

> Can not dump annotations for the task. Code: 500. Message: "FileNotFoundError: [Errno 2] No such file or directory: '/home/django/cvat/apps/annotation/cvat.py'\n"

with this log:
[cvat-log.txt](https://github.com/opencv/cvat/files/4409991/cvat-log.txt)

thanks again
